applications/vanilla/modules/class.discussionssortfiltermodule.php

```diff
@@ -197,8 +203,26 @@ protected function getFilterDropdowns() {
                     [],
                     ['rel' => 'nofollow']
                 );
+
+                $link = $dropdown->getItems()[val('group', $filter)]['items'][val('key', $filter)];
+                $group = val('group', $filter);
+                if($lastGroup != $group && $index != 0 && ($index != $filterLength - 1)) {
```

Comment:
Need a space between `if` and the opening parenthesis.

Reply:
`($index != $filterLength - 1)` would probably make more sense if it were written as: `$index != ($filterLength - 1)`.  The whole comparison doesn't need to be grouped in parenthesis, but it would be logical to group the subtraction expression.